docroot/themes/humsci/humsci_basic/templates/components/logo.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+  /**
+  * @file
+  * Logo Component
+  *
+  * The Stanford wordmark logo rendered with the custom Stanford ligature font.
```

Comment:
can you add a comment on if if this is also from stanford basic, and how it brings the logo I assume that's set somewhere in admin?

Reply:
please include this link! https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/dist/templates/decanter/components/logo/logo.twig

Reply:
I think logo is kinda misleading. It is the logo _but_ this component is not a logo in the traditional sense of being an image. It feels more like a wordmark. :smile: